recipes/magnum/all/conanfile.py

```diff
@@ -0,0 +1,710 @@
+                              "EGL_FOUND",
+                              "egl_system_FOUND")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "EGL::EGL",
+                              "egl::egl")
+
+    def build(self):
+        self._patch_sources()
+
+        cm = self._configure_cmake()
+        cm.build()
+
+    def package(self):
+        cm = self._configure_cmake()
+        cm.install()
+
+        build_modules_folder = os.path.join(self.package_folder, "lib", "cmake")
+        os.makedirs(build_modules_folder)
+        for exec in self._executables:
+            build_module_path = os.path.join(build_modules_folder, "conan-magnum-{}.cmake".format(exec))
+            with open(build_module_path, "w+") as f:
+                f.write(textwrap.dedent("""\
+                    if(NOT TARGET Magnum::{exec})
+                        if(CMAKE_CROSSCOMPILING)
+                            find_program(MAGNUM_EXEC_PROGRAM magnum-{exec} PATHS ENV PATH NO_DEFAULT_PATH)
+                        endif()
+                        if(NOT MAGNUM_EXEC_PROGRAM)
+                            set(MAGNUM_EXEC_PROGRAM "${{CMAKE_CURRENT_LIST_DIR}}/../../bin/magnum-{exec}")
+                        endif()
+                        get_filename_component(MAGNUM_EXEC_PROGRAM "${{MAGNUM_EXEC_PROGRAM}}" ABSOLUTE)
+                        add_executable(Magnum::{exec} IMPORTED)
+                        set_property(TARGET Magnum::gl-info PROPERTY IMPORTED_LOCATION ${{MAGNUM_EXEC_PROGRAM}})
+                    endif()
+                """.format(exec=exec)))
+
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        self.copy("*.cmake", src=os.path.join(self.source_folder, "cmake"), dst=os.path.join("lib", "cmake"))
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "Magnum"
+        self.cpp_info.names["cmake_find_package_multi"] = "Magnum"
+
+        magnum_plugin_libdir = "magnum-d" if self.settings.build_type == "Debug" else "magnum"
+        lib_suffix = "-d" if self.settings.build_type == "Debug" else ""
+
+        # The FindMagnum.cmake file provided by the library populates some extra stuff
+        self.cpp_info.components["_magnum"].build_modules.append(os.path.join("lib", "cmake", "conan-magnum-vars.cmake"))
+
+        # Magnum contains just the main library
+        self.cpp_info.components["magnum_main"].names["cmake_find_package"] = "Magnum"
+        self.cpp_info.components["magnum_main"].names["cmake_find_package_multi"] = "Magnum"
+        self.cpp_info.components["magnum_main"].libs = ["Magnum{}".format(lib_suffix)]
+        self.cpp_info.components["magnum_main"].requires = ["_magnum", "corrade::utility"]
+
+        # Audio
+        if self.options.with_audio:
+            raise Exception("Component not created")
```

Comment:
Is this a FIXME?

Reply:
I haven't build the library with this option activated. It requires `OpenAL` and I didn't realize it was already available. Letś see if it works.

And yes, this is like a TODO (same for other libraries for iOS, emsdk or Android): if someone manages to build them, they will need to populate the required component.

Reply:
Can you please output a nice message + explicit `FIXME` comment then?